src/script_chainer/win_exe/script_runner.py

```
@@ -419,32 +460,59 @@ def run_script(
-            return
-
-        print_message(f'脚本子进程创建成功 {script_path}', level='PASS')
+            print_message(f'脚本子进程创建成功 {script_path}', level='PASS')
 
-        # 3. 监控脚本运行状态
-        _monitor_script_done(script_config, state)
+            # 3. 监控脚本运行状态
+            try:
+                _monitor_script_done(script_config, state, last_log_time=last_log_time)
+            except _NoLogTimeoutError:
+                # 静默超时：终止当前进程，进入下一轮重试
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)